pkg/client/selfhosted/selfhosted.go

```diff
@@ -237,6 +238,7 @@ func (c *Client) doRequest(ctx context.Context, url, header string, obj interfac
 
 	if len(header) > 0 {
 		req.Header.Set("Accept", header)
+		req.Header.Add("Accept", ociV1Header)
```

Comment:
Should this be a comma-separated list? https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.1
```suggestion
		req.Header.Set("Accept", strings.Join(",", []string{header,ociV1Header})
```